api/src/main/java/jakarta/data/page/PageRequest.java

```diff
@@ -415,6 +443,28 @@ static <T> PageRequest<T> ofSize(int maxPageSize) {
      */
     PageRequest<T> sortBy(Sort<? super T> sort1, Sort<? super T> sort2, Sort<? super T> sort3, Sort<? super T> sort4, Sort<? super T> sort5);
 
+    /**
+     * Returns an otherwise-equivalent page request with
+     * {@link #requestTotal()} set to {@code false}, so that
+     * totals will not be retrieved from the database. Note
+     * that when totals are not retrieved by a repository
+     * method with return type {@link Page}, the operations
+     * {@link Page#totalElements()} and {@link Page#totalPages()}
+     * throw an {@link IllegalArgumentException} when called.
```

Comment:
The JavaDoc for the other methods states IllegalStateException.
```suggestion
     * throw an {@link IllegalStateException} when called.
```

Reply:
Oops 🫣that is definitely what I meant.